### ISSUE TITLE
📚 Documentation: [Apple SDK] `account.CreateOAuth2Session` is not `async`.

### ISSUE THREAD

**Issue by ItzNotABug** [appwrite/appwrite#6207]
### 💭 Description

The example on Apple SDK :
```swift
let success = try await account.createOAuth2Session(provider: "amazon")
```

However the method signature for `1.3x` & `1.4x` is same and hence not `async`.
See: [Account#createOAuth2Session](https://github.com/appwrite/sdk-for-apple/blob/4478da26033097be8ee13db501bf022d19d60b5a/Sources/Appwrite/Services/Account.swift#L1001C5-L1006C21).

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
@ItzNotABug thanks for creating this issue! 🙏🏼 Let me check with the team about this.

**Comment by abnegate** [comment]
@ItzNotABug This will become `async` again as a part of the fix for https://github.com/appwrite/appwrite/issues/6206

**Comment by abnegate** [comment]
Closing as fixed in Apple SDK 4.0.1
